.gitignore

```diff
@@ -1,16 +1,105 @@
-.idea/
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
 
-*.pyc
-*.ipynb
+# C extensions
+*.so
 
+# Distribution / packaging
+.Python
 build/
+develop-eggs/
 dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
 *.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
 
-.DS_Store
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
 
-# ducktape
-.ducktape/
-*results/
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
 
+# Unit test / coverage reports
+htmlcov/
+textcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
```

Comment:
i don't care about much stuff that's super-generic for python, but we have no django, jupyter, celery, sage, etc stuff. we should add stuff to `.gitignore` when we have good reason, not if there's an off-chance we might hit it...

Reply:
It's just the latest template from GitHub. We can choose and pick, but that's just more work / easier to use the latest template to cover the new stuff from tox and coverage reports, etcs.